openedx/features/partners/views.py

```diff
@@ -1,27 +1,60 @@
+import analytics
+import json
```

Comment:
The way I follow import statements order is:
```
python preinstalled libraries like re, json etc

pip installed packages like django, scrapy etc

project local module imports like from student.models import User etc.

```
Can you review if this order may be followed here?